locksmith/src/utils/certification.tsx

```diff
@@ -0,0 +1,104 @@
+import normalizer from './normalizer'
+import config from '../config/config'
+import { generateKeyMetadata } from '../operations/metadataOperations'
+import { Certificate } from '@unlock-protocol/ui'
+
+import { SubgraphKey } from '@unlock-protocol/unlock-js'
+import { addressMinify } from './strings'
+import satori from 'satori'
+import dayjs from 'dayjs'
+
+import { readFileSync } from 'fs'
+const inter400 = readFileSync('src/fonts/inter-400.woff')
+const inter700 = readFileSync('src/fonts/inter-700.woff')
+
+export const MAX_UINT =
+  '115792089237316195423570985008687907853269984665640564039457584007913129639935'
```

Comment:
move it to a separate file? I thought this was already a constant.

Reply:
not in locksmith, only in `unlock-app`, maybe we can add a shared lib later